openduck-py/openduck_py/routers/voice.py

```diff
@@ -136,31 +146,48 @@ async def start_response(
         await log_event(db, self.session_id, "started_response", audio=audio_data)
         self.is_responding = True
 
-        def _inference(sentence: str):
-            audio_chunk = styletts2_inference(text=sentence)
-            
-            audio_chunk_bytes = np.int16(audio_chunk * 32767).tobytes()
+        def _inference(sentence: str, audio_format: str = "int16"):
+            audio_chunk = styletts2_inference(
+                text=sentence,
+                output_sample_rate=self.output_sample_rate,
+            )
+            if audio_format == "int32":
```

Comment:
Can we cast the int32 to int16 on the client before sending it to the server?

Reply:
549cfbefe360dc9f5cd253e1c17f55a0363ef3bd

Reply:
Done here! ^